examples/Kindle2Folder.rb

```diff
@@ -0,0 +1,29 @@
+require 'kindle_highlights'
+
+def sanitize_filename(filename) #Function to remove invalid symbols from filenames
+  fn = filename.split /(?<=.)\.(?=[^.])(?!.*\
+  fn.map! { |s| s.gsub /[^a-z0-9\-]+/i, '_' }
+  return fn.join '.'
+end
+
+kindle = KindleHighlights::Client.new(
+  email_address: "aaa@aaa.com",
+  password: "password"
+  )
+
+folder = File.expand_path('~/Kindle') #Folder you want to store the txt files (in markdown format) in...
+
+kindle.books.each do |book|
+  fname = folder+"/"+sanitize_filename(book.title)+".txt"
+  print book.title+"\r\n" #Print name of Book to console so you can track progress
+  bookfile = File.open(fname, "w")
+	bookfile.puts "# #{book.title} by #{book.author} \r\n" #Header of the Book Title and Author
```

Comment:
Again, another nit-picky thing, but I'd like to see two spaces per level of indentation. I don't think this line needs to be indented at all.